troposphere/route53.py

```diff
@@ -77,11 +77,25 @@ class RecordSetGroup(AWSObject):
     }
 
 
+class AlarmIdentifier(AWSProperty):
+    props = {
+        'Name': (basestring, True),
+        'Region': (basestring, True),
+    }
+
+
 class HealthCheckConfiguration(AWSProperty):
     props = {
+        'AlarmIdentifier': (AlarmIdentifier, False),
+        'ChildHealthChecks': (basestring, False),
```

Comment:
I have a question out to the CF team about ChildHealthChecks as the docs implied it was a list of strings but the json/yaml indicated just a single string.